tests/fiaas_deploy_daemon/test_e2e.py

```diff
@@ -177,6 +177,12 @@ def ready():
                 Ingress: "e2e_expected/tls-ingress-cert-issuer.yml",
                 HorizontalPodAutoscaler: "e2e_expected/tls-hpa-cert-issuer.yml",
             }),
+            ("v3/data/examples/tls_enabled_multiple.yml", {
+                Service: "e2e_expected/tls-service-multiple.yml",
+                Deployment: "e2e_expected/tls-deployment-multiple.yml",
```

Comment:
This file seems to be missing.

Reply:
It is not missing, but I have apparently missed deleting the service,deployment and hpa entries in this test case, fixing it now